### ISSUE TITLE
methcoverage only shows one dot

### ISSUE THREAD

**Issue by RRebo** [xie186/ViewBS#60]
Hi,
I have used methcoverage before, and it did work well. Now, with a new set of data, while all the other commands are working, I get a one dot graph for each of my samples on the methcoverage graph. I added the metcoverage folder created during the run and here is one of my CG_report files: https://drive.google.com/file/d/1BumWLr8brd4sPSx6OVRk37tfuAlAnacK/view?usp=sharing

`~/ViewBS-0.1.9/ViewBS MethCoverage --reference Hd_genome_1.0.fasta --sample St-1-calyx-60-d_S4_R1_001_val_1_bismark_bt2_pe.sorted.deduplicated.CpG_report.txt.gz,St1calyx60d --sample St-1-calyx-57-e_R1_001_val_1_bismark_bt2_pe.sorted.deduplicated.CpG_report.txt.gz,St1Calyx57e --sample St-1-calyx-63-f_R1_001_val_1_bismark_bt2_pe.sorted.deduplicated.CpG_report.txt.gz,St1Calyx63f --sample St-1-thorax-5-j_R1_001_val_1_bismark_bt2_pe.sorted.deduplicated.CpG_report.txt.gz,St1Thorax5j --sample St-3-calyx-37-b_R1_001_val_1_bismark_bt2_pe.sorted.deduplicated.CpG_report.txt.gz,St3Calyx37b --sample St-3-calyx-58-a_R1_001_val_1_bismark_bt2_pe.sorted.deduplicated.CpG_report.txt.gz,St3Calyx58a --sample St-3-calyx-60-c_R1_001_val_1_bismark_bt2_pe.sorted.deduplicated.CpG_report.txt.gz,St3Calyx60c --sample St-3-thorax-5-g_R1_001_val_1_bismark_bt2_pe.sorted.deduplicated.CpG_report.txt.gz,St3Thorax5g --sample St-3-thorax-5-h_R1_001_val_1_bismark_bt2_pe.sorted.deduplicated.CpG_report.txt.gz,St3Thorax5h --sample St-3-thorax-5-i_R2_val_1_bismark_bt2_pe.sorted.deduplicated.CpG_report.txt.gz,St3Thorax5i --outdir metCoverage --prefix EpiHypo`

[ViewBS.zip](https://github.com/xie186/ViewBS/files/6569833/ViewBS.zip)

Thank you
Rita

**Comment by Kennyluo4** [comment]
I have similar issue. Seems it adds up the CG sites from all samples as the denominator. I got over 70% sites with at least 1 coverage when plotting just 1 sample. It dropped to less than 30% when I submitted 3 samples. And I got just 1 dot for each sample when using 9 samples.  
MethCoverage plot is very confusing, the histogram output from Methylkit is more straightforward.
<img width="740" alt="image" src="https://user-images.githubusercontent.com/16495389/120906572-a1c58500-c628-11eb-98d0-b97ef55843ab.png">
<img width="740" alt="image" src="https://user-images.githubusercontent.com/16495389/120906580-aee27400-c628-11eb-8034-d48942e46962.png">
